lib/mempool/mempool.js

```diff
@@ -1233,6 +1254,17 @@ class Mempool extends EventEmitter {
     if (!proof.isSane())
       throw new VerifyError(proof, 'invalid', 'bad-aidrop-proof', 100);
 
+    const key = proof.getKey();
+
+    if (!key)
+      throw new VerifyError(proof, 'invalid', 'bad-airdrop-proof', 100);
+
+    if (this.network.airdrop.disableGoosig) {
+      if (this.chain.height >= this.network.airdrop.disableGoosigHeight
```

Comment:
I just realized in the mempool all the height checks are `chain.height + 1` -- as in, the mempool checks what is valid in the _next_ block. I gotta fix this in #330 as well

Reply:
Good looks